passkeeper/tests/units/test_passkeeper.py

```diff
@@ -64,16 +64,17 @@ def test_init_dir(self, mock_create_dir, mock_encrypt, mock_cleanup):
         mock_cleanup.assert_called_once_with()
 
 
-    @patch('passkeeper.Passkeeper._remove_old_encrypted_files')
+    @patch('passkeeper.Passkeeper.remove_old_encrypted_files')
     @patch('passkeeper.run_cmd')
     @patch('passkeeper.os.listdir')
     @patch('passkeeper.os.path.isfile')
-    def test_cleanup_ini(self, mock_isfile, mock_listdir, mock_cmd,
-                               mock_remove_old_encrypted_files):
+    def test_cleanup(self, mock_isfile, mock_listdir, mock_cmd,
+                     mock_remove_old_encrypted_files):
         # One ignored file and one valid file
         mock_listdir.return_value = ['ignored', 'bar.ini']
         mock_isfile.return_value = True
-        self.pk.cleanup_ini()
+        self.pk.remove_old_encrypted_files(force_remove=False)
```

Comment:
This test is only for cleanup, we don't want to test remove anymore in this test.
Remove is now tester in `test_remove_old_encrypted_files`